tools/build-help-center

```diff
@@ -19,12 +19,11 @@ from django.conf import settings
 
 
 def run() -> None:
-    subprocess.check_call(["./tools/convert-help-center-docs-to-mdx"], stderr=subprocess.STDOUT)
+    subprocess.check_call(["./tools/convert-help-center-docs-to-mdx"])
     env = os.environ.copy()
     env["SHOW_BILLING_HELP_LINKS"] = str(settings.CORPORATE_ENABLED).lower()
     subprocess.check_call(
         ["/usr/local/bin/corepack", "pnpm", "build"],
-        stderr=subprocess.STDOUT,
```

Comment:
`STDOUT` is definitely not the default; I'm pretty sure this makes it go to `STDERR` instead, which is probably correct, but the commit message is wrong.